doc/api/single-executable-applications.md

```diff
@@ -54,13 +54,13 @@ tool, [postject][]:
 
    Using PowerShell:
 
-   ```bash
+   ```powershell
    cp (Get-Command node).Source hello.exe
    ```
 
    Using Command Prompt:
 
-   ```bash
+   ```text
```

Comment:
```suggestion
   ```cmd
```
Shouldn't this be cmd?

Reply:
The linter won’t let me do that.

Reply:
Okay, so here's the total list:
https://github.com/nodejs/node/blob/260092eca316bae6036e52b2cdc73c4c4f8a9da6/tools/lint-md/lint-md.mjs#L20828-L20843
I think `console` still makes more sense than `text` here.

Reply:
I disagree, `console` is meant for showing the output of commands (with optionally the command themselves when prefixed with `$` or `>`), so it wouldn’t apply here. If you feel strongly that `text` is not correct, let’s add `cmd` to the list.

Reply:
I don't feel strongly about it :)